bubblesub/cmd/file_ops.py

```diff
@@ -312,6 +334,13 @@ class UnloadAudioCommand(BaseCommand):
     names = ["unload-audio"]
     help_text = "Unloads currently loaded audio file."
 
+    @property
+    def is_enabled(self) -> bool:
+        return (
+            self.api.gui.current_view == ViewState.Audio
+            or self.api.gui.current_view == ViewState.Full
+        )
```

Comment:
This is a conceptual fail. First of all I should be able to load / unload streams regardless of whether the widgets for them are shown or not, otherwise you're limiting users for no good reason. Also, as it is, if I load a video and switch to compact view, the stream stays loaded, but I suddenly can't unload it?…

Reply:
Since these commands aren't related to playback, I haven't changed my code here. Before to update the other ones, I need some advice about this choice.